Rakefile

```diff
@@ -23,12 +24,15 @@ Thread.abort_on_exception = true
 
 begin
   require 'rspec/core/rake_task'
+  require 'rubocop/rake_task'
+
+  RuboCop::RakeTask.new
 
   desc "Run specs"
   RSpec::Core::RakeTask.new(:spec) do |t|
-    t.rspec_opts = %w(-fs --color)
-    #t.ruby_opts  = %w(-w)
+    t.rspec_opts = %w(--color)
   end
+  task :spec => :rubocop
   task :default => :spec
```

Comment:
But then `rubocop` task won't be invoked when you run `spec`. That's not what @segiddins wanted.